### DISCUSSION TITLE
Week 6

### DISCUSSION THREAD

**Discussion by Stev3eM** [SE-TINF22B6/CookHub#47] [Show and tell]
Hello backpackers,

This is our weekly discussion-blog to keep you up to date within our project progression.
We're already in week six and have a view updates to present:
<br>

 ### Slight adjustments to project structure and documentation updates
------

_Therefore we have_

✅ combined the Like-Function and the Favorite-Function instead of handleing them partially
✅ created  the [CRC diagram](https://github.com/SE-TINF22B6/CookHub/blob/main/Project_Doc/Diagrams/CRC_CH_bluePrint.png)
✅ created an [ER/EER diagram](https://github.com/SE-TINF22B6/CookHub/blob/main/Project_Doc/Diagrams/er-diagram.drawio.svg)
✅ created [class diagrams](https://github.com/SE-TINF22B6/CookHub/blob/main/Project_Doc/Diagrams/uml-class-diagram.drawio.svg) out of CRC
✅ adjusted [login-sequence-diagram](https://github.com/SE-TINF22B6/CookHub/blob/main/Project_Doc/Diagrams/login-sequence-diagram.drawio.svg) and [openai-sequence-diagram](https://github.com/SE-TINF22B6/CookHub/blob/main/Project_Doc/Diagrams/openai-sequence-diagram.drawio.svg) to match with the class diagrams
✅ created brief explanation of the classes and their relationships
<br>

### Initial testing of prompt-settings for ChatGPT to wrap up a recipe to an adventure story
------

✅ [Test 01 - ChatGPT_Answer](https://github.com/SE-TINF22B6/CookHub/assets/113697513/35f57aec-fe42-46b6-b9e4-7cf52dc90704)

_DevelopersOpinion:_ At first glance the answer might be very ok for that shortly given prompt. WIth more of these tests in the upcomming weeks we will figure out what's the best prompt or if it's possible at a consistent level to satisfy our customer demands in the first place. The answer could have get a big impact within our developer group. We keep you up to date!

<br><br>

See you next time!

Your CookHub Team
<img width="50" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/cdbd0688-3eb9-46fa-a608-baf2efbaba88">

**Comment by AndresMasis** [comment]
Hi

The diagrams you shared expose a clear path of your workflow. It allows an external person like me to understand how you are developing your software. Congratulations.

I think your achievment with the ChatGPT is very remarkable. It demonstrates solid progress in your project by adopting innovative technologies.

Cheers from CookPit.

**Comment by giomskii** [comment]
Good evening!

Your diagrams depict the entities, its responsibilities and their relations to each other in a clear way. Only in regards to your uml class 
diagram, I think it would have been better to add a cardinality ratio for a better understanding of the relationship. 
It's also nice to see that you have updated your diagrams/sequences from the last weeks and adjusted it to the newly created crc and class diagrams.  Moreover, which website did you use for your CRC-diagrams to identify your classes? 

I am very exited to see your progress with using ChatGPT to its fullest extent. 👍
